tests/fixtures/scenarios/exitCode/components/terraform/hook-and-store/main.tf

```diff
@@ -0,0 +1,34 @@
+terraform {
+  required_providers {
+    null = {
+      source  = "hashicorp/null"
+      version = "~> 3.0"
+    }
+  }
+}
+
+variable "stage" {
+  description = "Stage. Used to define an Atmos stack."
+  type        = string
+  default     = "test"
+}
+
+resource "null_resource" "run_once" {
+  # Use a changing trigger to force the provisioner to run every time.
+  triggers = {
+    always_run = timestamp()
+  }
+
+  provisioner "local-exec" {
+    interpreter = ["/bin/bash", "-c"]
```

Comment:
Yes, this is not a very cross platform friendly solution (bash is not always installed and requires WSL on windows). Let's use another strategy.

Consider maybe using input validation instead. 

https://developer.hashicorp.com/terraform/language/values/variables#custom-validation-rules

Reply:
Done.

Reply:
Removed this